packs/classfeatures/extensible-weapon.json

```diff
@@ -24,7 +24,17 @@
             "remaster": false,
             "title": "Pathfinder Guns & Gears"
         },
-        "rules": [],
+        "rules": [
+            {
+                "definition": [
+                    "item:id:{actor|flags.pf2e.trackedItems.weaponInnovation}"
+                ],
+                "key": "AdjustStrike",
+                "mode": "add",
+                "property": "weapon-traits",
+                "value": "reach"
+            }
+        ],
```

Comment:
We can also increase the reach by another 5 if the weapon has the reach trait. An AE like to add 5 to `system.attributes.reach.base`